code/modules/html_interface/voting/voting.dm

```diff
@@ -158,6 +167,7 @@ var/global/datum/controller/vote/vote = new()
 	return .
 
 /datum/controller/vote/proc/announce_result()
+	stack_trace("Fuck my shit up. Lock is \[[lock]]")
```

Comment:
Yes but BYOND's handling of exceptions is stupid.

If there is no try catch in the call stack the crash terminates the current proc and makes it return `.`, if there IS a try catch it unwinds the stack and runs the catch.